### PR TITLE
Active low

### PR DESCRIPTION
**Pull request by danielwu68**
Adding support for LED that turns on at digital low.

### CODE REVIEW

File: morse.h [hunk 1]
```diff
@@ -233,11 +233,20 @@ class SpeakerMorseSender: public MorseSender {
...
+	private:
+		bool activeLow;
 	protected:
 		virtual void setOn();
 		virtual void setOff();
 	public:
-		LEDMorseSender(int outputPin, float wpm=WPM_DEFAULT);
+		/**
+		 * Creates a LED Morse code sender with the given GPIO pin.  The optional
+		 * boolean activeLow indicates LED is ON with digital LOW value.
```
Comment (markfickett) [inline morse.h]: Thanks for adding a doc too! I think this line is covered by the @param below, so it's OK to omit so there's not duplication.

File: morse.h [hunk 2]
```diff
@@ -233,11 +233,20 @@ class SpeakerMorseSender: public MorseSender {
...
+		 * @param outputPin GPIO pin number
+		 * @param activeLow set to true to indicate the LED ON with digital LOW value.  default: false
+		 * @param wpm words per minute, default: WPM_DEFAULT
+		 */
+		LEDMorseSender(int outputPin, bool activeLow = false, float wpm=WPM_DEFAULT);
```
Comment (markfickett) [inline morse.h]: Won't this break existing usage of the wpm argument? (Does `examples/speeds/speeds.pde` still work?) What about making `activeLow` the third argument, and/or adding a `setActiveLow(boolean)`?

**Review by markfickett — COMMENTED** [review]
Looks handy, thanks!

Could you rebase this PR, now that I've merged in the type fixes?

**Comment by danielwu68** [comment]
I was debating which way to go as the SpeakerMorseSender has the wpm last but PWMMorseSender has wpm as the 2nd param.  I was thinking to force people to do it the same way but probably better not to break existing code.  

Finally decided to add a 2-param constructor.  I tested it with speeds.pde and seems to work.  Please test it out.

**Comment by markfickett** [comment]
Thanks for making the update! Merged the other PR. (Sorry it's taken a while to get to this!)

**Pull request closed without merging**